src/components/Link.tsx

```diff
@@ -28,6 +28,7 @@ type BaseProps = {
   isPartiallyActive?: boolean
   activeStyle?: StyleProps
   customEventOptions?: MatomoEventOptions
+  imageLink?: boolean
```

Comment:
Nit pick, but perhaps `isImageLink` to make the name clear it's a boolean and not a string to the image